client/src/components/Cart/Cart.js

```diff
@@ -73,20 +74,19 @@ const Cart = ({ match }) => {
   }, [value]);
 
   const fetchingProducts = async () => {
-    console.log(state.user);
-    const fetchedProduct = await axios.post(`${BASE_URL}/cart`, {
-      user: state.user
+    const fetchedProduct = await axios.post(`${BASE_URL}/cart`, state, {
```

Comment:
Why do we need to pass state here? I think we only need the user object from state

Reply:
okay doing the necessary changes